lib/routemap.h

```diff
@@ -241,6 +241,146 @@ struct route_map {
 };
 DECLARE_QOBJ_TYPE(route_map)
 
+/* Route-map match conditions */
+#define IS_MATCH_INTERFACE(C)                                                  \
+	(strncmp(C, "frr-route-map:interface", strlen(C)) == 0)
```

Comment:
More succinct to just convert all of these to
```
strmatch(C, "xpath")
```
I don't see the need for the case comparison as all xpaths should be lower case IMO